apps/client/src/client/views.clj

```diff
@@ -24,7 +24,7 @@
     [:header#top
      [:h1 [:a.home {:href "/"} "sharing.io"]]
      [:nav
-      (when permitted-member
+      (when (and permitted-member (not (empty? (:ssh-keys user))))(not= (:ssh-keys user) nil)
```

Comment:
In clojure, if you call `empty?` on a string, it will return true if string == `""` or string == `nil`. 
So this conditional could be simplified to:
`(when (and permitted-member (not (empty? (:ssh-keys user)))))`

However, there's an option to simplify it further using a clojure idiom. Instead o calling `(not (empty? val))` you can just say `(seq val)`, as it will return nil if the value is `""` or `nil`, and nil is a falsy value for the purpose of this conditional statement.

In that case, the code could be reduced to:
```
(when (and permitted-member (seq (:ssh-keys user))) 
...do-stuff)
```

Reply:
Oh! The `(not= (:ssh-keys user) nil)` might've been from when I was testing around things.

Testing it with `(seq ...)` works very well! Geez, that's so simple.

Reply:
Updated in c18eedb